config.js

```diff
@@ -2,25 +2,26 @@
 const chalk = require('chalk')
 const noop = x => x
 const list = {
-  fg: 'black',
-  bg: 'cyan',
+  fg: 'cyan',
```

Comment:
Styles can be changes via themes. See https://www.npmjs.com/package/fx-theme-monokai

Reply:
Yes, I've had trouble getting the theme to register for some reason. Apologies this landed in the PR, it should've stayed local.